ietf/nomcom/utils.py

```diff
@@ -612,6 +612,20 @@ def three_of_five_eligible(previous_five, queryset=None):
         queryset = Person.objects.all()
     return queryset.filter(meetingregistration__meeting__in=list(previous_five),meetingregistration__attended=True).annotate(mtg_count=Count('meetingregistration')).filter(mtg_count__gte=3)
 
+def new_three_of_five_eligible(previous_five, queryset=None):
+    """ Return a list of Person records who attended at least 
+        3 of the 5 type_id='ietf' meetings before the given
+        date. Does not disqualify anyone based on held roles.
+        This 'new' variant bases the calculation on the Meeting.Session model rather than Stats.MeetingRegistration
+        Leadership will have to create a new RFC specifying eligibility (RFC8989 is timing out) before it can be used.
+    """
+    if not queryset:
```

Comment:
Should test `is None` here (otherwise someone passing in a queryset that happened to be empty will be surprised to find it was ignored:
```
>>> not Person.objects.none()
True
```

Reply:
very good catch. Also fixed the previous function.